client/dashboard/sites/fields.tsx

```diff
@@ -129,6 +129,7 @@ const DEFAULT_FIELDS: Field< Site >[] = [
 		id: 'php_version',
 		label: __( 'PHP version' ),
 		render: ( { item }: { item: Site } ) => <PHPVersion site={ item } />,
+		enableSorting: false,
```

Comment:
I almost disabled sorting on the Hosting field too. It's one of the new fields that we ever used to sort on. So why enable it when so many other fields aren't sortable?

On the other hand, we know host will never be `undefined` because it defaults to `DEFAULT_PROVIDER_NAME`. So we can be sure it doesn't crash.